### PR TITLE
Improve view templates for scheduling an appointment

### PR DESCRIPTION
**Pull request by Guntrisoft**
**Fix typo in error message**

**Before**
screen shot 2016-11-22 at 17 12 55

**After**
screen shot 2016-11-22 at 17 04 18

**Reduce duplication across 'new' and 'reschedule' appointments**

screen shot 2016-11-22 at 17 06 53

### CODE REVIEW

**Comment by benbarnett** [comment]
Can we have screenshots and context please? ☺️

You know if you put a description in the commit it will populate the description of the PR automatically?

**Comment by Guntrisoft** [comment]
@benbarnett Added screenshots

File: app/views/appointments/new.html.erb [hunk 1]
```diff
@@ -14,20 +14,9 @@
     <div class="col-md-12">
       <h2>Choose available slot <%= required_label %></h2>
       <% if @appointment.errors[:guider].any? %>
```
Comment (AndrewVos) [inline app/views/appointments/new.html.erb]: Tempting to move this `if` statement into the partial maybe?

**Comment by AndrewVos** [comment]
Sorry can't submit single comments on my phone so merge if you wanna ignore that one comment ^^

File: app/views/appointments/new.html.erb (hunk 1 above)
Comment (Guntrisoft) [inline app/views/appointments/new.html.erb]: Yup - good point, have moved the `h2` aswell into the partial

**Comment by Guntrisoft** [comment]
@AndrewVos ok with the change?

**Pull request merged**